src/plugins/codeStyle/index.spec.ts

```diff
@@ -375,6 +375,22 @@ describe('codeStyle', () => {
         expect(actual).deep.equal(expected);
     });
 
+    it('enforce no stop', async () => {
+        const diagnostics = await linter.run({
+            ...project1,
+            files: ['source/no-stop.brs'],
+            // Should be warn by default
```

Comment:
Are there any example tests on how to enforce this?

Reply:
What do you mean "enforce"?

Reply:
How can I ensure that a setting is "warn" by default. Are there any tests that showcase that?